### DISCUSSION TITLE
Avoiding loading screen with custom splash screen

### DISCUSSION THREAD

**Discussion by mad4gith** [flyinghead/flycast#968] [Ideas]
Hi,
is there a chance to have loading hided or showing just the game box art while starting a game?

Thank you.

**Comment by flyinghead** [comment]
It's not possible currently. I don't understand why people dislike having a loading screen so much.

Some Naomi games take up to 10 seconds or more to load. If the application doesn't update the screen for that long, it's at risk of being killed on Android (ANR). So it's both useful to show the user that Flycast is busy, and to the operating system to indicate that the application is responding.

**Reply by mad4gith** [reply]
I've installed Flycast on my cab. I just think it ruin the "magic" like that MAME nag messages.
It's just an opinion.
Sorry.

**Reply by zaphod4254** [reply]
Same.  When using in a cabinet with a front end, the loading screen kind of spoils the "feel" of the otherwise seamless interface.
